common/logisticspipes/gui/GuiChassiPipe.java

```diff
@@ -209,9 +178,15 @@ private String getModuleName(int slot) {
 
 	@Override
 	protected void drawGuiContainerBackgroundLayer(float f, int x, int y) {
-		GL11.glColor4f(1.0F, 1.0F, 1.0F, 1.0F);
-		mc.renderEngine.bindTexture(_chassiPipe.getChassiGUITexture());
-		drawTexturedModalRect(guiLeft, guiTop, 0, 0, xSize, ySize);
+		GuiGraphics.drawGuiBackGround(mc, guiLeft, guiTop, right, bottom, zLevel, true);
+
+		GL11.glTranslated(guiLeft, guiTop, 0);
```

Comment:
Please don't use GL11 methods here, just put "guiLeft + 17" instead of 17 below, and so on...